authentik/lib/default.yml

```diff
@@ -105,19 +105,14 @@ reputation:
 cookie_domain: null
 disable_update_check: false
 disable_startup_analytics: false
-avatars: env://AUTHENTIK_AUTHENTIK__AVATARS?gravatar,initials
 geoip: "/geoip/GeoLite2-City.mmdb"
 
-footer_links: []
-
-default_user_change_name: true
-default_user_change_email: false
-default_user_change_username: false
-
-gdpr_compliance: true
 cert_discovery_dir: /certs
 default_token_length: 60
-impersonation: true
+
+tenants:
+  enabled: false
+  api_key: ""
```

Comment:
I'm still trying to figure out a better way of dealing with the tenant api key instead of having it in the config

Reply:
I think this is fine for a first iteration. In the longer run, we can have an `/if/tenants` frontend, that is only accessible by superusers in the default tenant, same for the API.